src/kv/store.h

```diff
@@ -694,12 +712,15 @@ namespace kv
 
       {
         std::lock_guard<SpinLock> vguard(version_lock);
-        // This can happen when a transaction started before a view change,
-        // but tries to commit after the view change is complete.
-        LOG_DEBUG_FMT(
-          "Want to commit for term {}, term is {}", txid.term, term);
         if (txid.term != term)
+        {
+          // This can happen when a transaction started before a view change,
```

Comment:
This was always displayed. I think we only want to display this message if a transaction fails because the term has changed between the time the opacity version was taken and the transaction is committed (i.e. here).

Reply:
It's a debug/diag, it's not really useful in normal operation.